rd-net/RdFramework/Impl/SocketWire.cs

```diff
@@ -112,8 +112,8 @@ protected Base(string id, Lifetime lifetime, [NotNull] IScheduler scheduler) : b
       private Timer StartHeartbeat()
       {
         void OnTimedEvent(object sender, ElapsedEventArgs e)
-        {
-          Ping();
+        {
+          Scheduler.Queue(Ping);
```

Comment:
Using the default scheduler here may lead to wire being marked as potentially disconnected in cases of busy scheduler. With the old semantic pings were always sent regardless of what other threads were doing. Perhaps a ping scheduler should be added?

Reply:
I just restored the behavior that existed before my changes. I introduced that Scheduler and then removed